example/README.md

```diff
@@ -0,0 +1,57 @@
+# Docker Compose example
+
+This folder contains a Docker Compose example for the Snowplow realtime pipeline.
+
+## Introduction
+
+This Docker Compose example bundles the following components in 4 distinct containers:
```

Comment:
4 containers but only two listed here?

Reply:
oops yes, I originally wanted to provide an example with s3 loader and es loader but since the es domain in aws can't be set fully (i.e. depends on your account) I decided to limit myself to ssc and se.